### DISCUSSION TITLE
Version 2.6.0

### DISCUSSION THREAD

**Discussion by BourgeoisLab** [kleiner-brauhelfer/kleiner-brauhelfer-2#304] [General]
- Neu: Umstellung auf **Qt v6.5**
- Neu: Datenbankversion **2009**
- Neu: Knopf, um Sudhausausbeute anhand der Malzgaben anzupassen
- Neu: Diagramme überarbeitet (neu mit QCustomPlot statt Qt Charts)
- Neu: Eingabemöglichkeit einer Stammwürze für das Verschnittwasser
- Neu: Maischplan überarbeitet:
  - Zuschütten ermöglicht Schüttungsgabe zu einem späteren Zeitpunkt
  - Gesamtwassermenge (Einmaischen+Zubrühen) muss Hauptgussmenge entsprechen
  - Möglichkeit Wassermengen auszugleichen oder Hauptgussmenge anzupassen
  - Möglichkeit Schüttungsmengen auszugleichen
- Neu: Hefestarter lässt sich im Reiter Braudaten verändern.
- Fix: Gaben & Rasten im Rezeptreiter nicht immer aktualisiert 
- Fix: Anzeige der Hopfengaben mit Kochdauer von 0 Minuten im Spickzettel

<hr /><em>This discussion was created from the release <a href='https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/tag/v2.6.0'>Version 2.6.0</a>.</em>

**Comment by Rasinari** [comment]
"Neu: Diagramme überarbeitet (neu mit QCustomPlot statt Qt Charts)"

Schrift schiebt sich bei mehreren Einträgen oder langen Namen übereinander. Unleserlich.
Vorschlag: Zeilenumbruch einfügen.

**Comment by hugobrew** [comment]
bei mir startet der Brauhelfer nach dem update unter windows 10 nicht mehr...

**Reply by BourgeoisLab** [reply]
Unter C:\Program Files\kleiner-brauhelfer-2 alle Dateien löschen, die mit Qt5 anfange und nochmals versuchen. Alternativ Programm deinstallieren, sicherstellen, dass der Ordner C:\Program Files\kleiner-brauhelfer-2 leer ist und das Programm neu installieren.

Einstellungen und Datenbank gehen nicht verloren, sofern man nicht explizit den Ordner als Speicherort für die Datenbank angegeben hat.

**Comment by markussoeth** [comment]
Danke für das Update und die damit verbundene Arbeit!
Sieht wiedermal Klasse aus!
